packages/fx-core/templates/plugins/resource/apiconnector/sample/js/cert.template

```diff
@@ -4,7 +4,7 @@ const teamsfxSdk = require("@microsoft/teamsfx");
 const teamsFx = new teamsfxSdk.TeamsFx();
 // Initializes a new axios instance to call {{config.APIName}} API.
 const authProvider = new teamsfxSdk.CertificateAuthProvider(
-  // Please add code to read your cert and private key, replace "<your-cert>" and "<your-private-key>" with actual value.
+  // TODO: Please add code to read your cert and private key, replace "<your-cert>" and "<your-private-key>" with actual value.
```

Comment:
TODO: Add code to read your certificate and private key. Replace "<your-cert>" and "<your-private-key>" with your actual certificate and private key values.